mlrun/api/utils/projects/member.py

```diff
@@ -147,3 +149,36 @@ def get_project_owner(
         name: str,
     ) -> mlrun.api.schemas.ProjectOwner:
         pass
+
+    async def on_after_delete_project(
```

Comment:
just to follow mlrun convention
```suggestion
    async def post_delete_project(
```